ui/page/transaction/transactions_page.go

```diff
@@ -209,99 +276,159 @@ func (pg *TransactionsPage) loadTransactions(offset, pageSize int32) ([]*sharedW
+		layoutContents = append(layoutContents, layout.Stacked(func(gtx C) D {
+			gtx = gtx.Disabled()
+			overlayColor := pg.Theme.Color.Gray3
+			overlayColor.A = 220
+			gtx.Constraints.Min.X = gtx.Constraints.Max.X
+			gtx.Constraints.Min.Y = gtx.Constraints.Max.Y - gtx.Dp(values.MarginPadding60)
+			cryptomaterial.FillMax(gtx, overlayColor, 10)
+
+			lbl := pg.Theme.Label(values.TextSize20, values.String(values.StrFunctionUnavailable))
+			lbl.Font.Weight = font.SemiBold
+			lbl.Color = pg.Theme.Color.PageNavText
+			return cryptomaterial.CentralizeWidget(gtx, lbl.Layout)
+		}))
+	}
+
+	return layout.Stack{Alignment: layout.NW}.Layout(gtx, layoutContents...)
+}
 
-	items := []layout.FlexChild{layout.Rigid(pg.pageTitle)}
-	if pg.WL.SelectedWallet.Wallet.GetAssetType() == utils.DCRWalletAsset {
-		// Layouts only supportted by DCR
-		line := pg.Theme.Separator()
-		line.Color = pg.Theme.Color.Gray3
-		items = append(items, layout.Rigid(line.Layout))
-		items = append(items, layout.Rigid(pg.sectionNavTab))
+func (pg *TransactionsPage) layoutDesktop(gtx C) D {
+	items := []layout.FlexChild{}
+	if pg.selectedWallet == nil || pg.selectedWallet.GetAssetType() == utils.DCRWalletAsset {
+		// Only show tx category navigation txCategoryTab for DCR wallets.
+		items = append(items, layout.Rigid(pg.txCategoriesNav))
+	}
+
+	items = append(items, layout.Rigid(pg.desktopLayoutContent))
+	if pg.isHomepageLayout {
+		return components.UniformPadding(gtx, func(gtx C) D {
+			return layout.Flex{Axis: layout.Vertical}.Layout(gtx, items...)
+		})
 	}
-	items = append(items, txlisingView)
 	return layout.Flex{Axis: layout.Vertical}.Layout(gtx, items...)
 }
 
-func (pg *TransactionsPage) layoutMobile(gtx layout.Context) layout.Dimensions {
-	wal := pg.WL.SelectedWallet.Wallet
+func (pg *TransactionsPage) txCategoriesNav(gtx C) D {
+	return cryptomaterial.CentralizeWidget(gtx, func(gtx C) D {
+		return layout.Inset{Bottom: values.MarginPadding16}.Layout(gtx, pg.txCategoryTab.Layout)
+	})
+}
+
+func (pg *TransactionsPage) desktopLayoutContent(gtx C) D {
+	if pg.walletNotReady() && pg.walletDropDown == nil {
+		return pg.txListLayout(gtx) // nothing else to display on this page at this time
+	}
+
+	pageElements := []layout.StackChild{layout.Expanded(pg.txListLayout)}
+
+	if pg.walletDropDown != nil { // display wallet dropdown if available
+		walletDropdownWidget := layout.Expanded(func(gtx C) D {
+			return pg.walletDropDown.Layout(gtx, 0, false)
+		})
+		pageElements = append(pageElements, walletDropdownWidget)
+	}
+
+	// display tx dropdown if selected wallet is ready and showLoader is false
+	if !pg.walletNotReady() && !pg.showLoader {
+		pg.ParentWindow().Reload() //refresh UI to display updated txType dropdown
+
+		txDropdownWidget := layout.Expanded(func(gtx C) D {
+			return pg.txTypeDropDown.Layout(gtx, 0, true)
+		})
+		pageElements = append(pageElements, txDropdownWidget)
+	}
+
+	return layout.Stack{Alignment: layout.NW}.Layout(gtx, pageElements...)
```

Comment:
Another scenario where there could be only one item in `pageElements` and wrapping it in a stack is unnecessary.
Also, not sure the alignment is necessary.